test/inductor/test_torchinductor_opinfo.py

```diff
@@ -443,18 +458,22 @@ def test_comprehensive(self, device, dtype, op):
             self.skipTest(f"{op_name} in {dtype} not supported")
         elif dtype in inductor_expected_failures_single_sample[device_type].get(
             op_name, set()
+        ) or dtype in inductor_gradient_expected_failures_single_sample[
```

Comment:
If there's a large list of grad failures, we should consider splitting the test into forward or exclude, and then backward or exclude (instead of forward+backward or exclude from either). So as not to regress forward tests.

Reply:
We have 31 new ops failing due to gradient check... which is manageable...